web/src/tools/testing/test-utils/build.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+#
+# Compile KeymanWeb's test-utils module
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../../../resources/build/builder.inc.sh"
+## END STANDARD BUILD SCRIPT INCLUDE
+
+SUBPROJECT_NAME=tools/testing/test-utils
+. "$KEYMAN_ROOT/web/common.inc.sh"
+. "$KEYMAN_ROOT/resources/shellHelperFunctions.sh"
+
+################################ Main script ################################
+
+builder_describe "Builds the Keyman Engine for Web's test-utils module" \
```

Comment:
This should instead say what the test-utils module is (because the path already tells us that's it is web's test-utils module)

```suggestion
builder_describe "Automated js/ts test utilities for KeymanWeb" \
```